ext/geometry/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+[package]
+name = "deno_geometry"
+version = "0.1.0"
+authors.workspace = true
```

Comment:
Note to self: I already claimed https://crates.io/crates/deno_geometry to avoid problems during publishing when this PR lands